### DISCUSSION TITLE
WTF, Move dependencies is going to charge gases by code bytes!

### DISCUSSION THREAD

**Discussion by ghost** [aptos-labs/aptos-developer-discussions#211] [General]
Recently I found my txn gas cost being high than before, then I try to gas profiling then I found, WTF Move dependencies begin to count to charge gas! That is not happened before months ago.

**Comment by ghost** [comment]
https://governance.aptosfoundation.org/proposal/69

**Comment by davidiw** [comment]
Apologies, there was an oversight, where we anticipated sharing [this](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-84.md) AIP as the update happened. Would be happy to receive other developers feedback either here or on the AIP itself.
